nodetypes/https%3A%2F%2Fust-quantil.github.io%2Fnodetypes/QiskitRuntime_w1/NodeType.tosca

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:winery="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypes" id="nodeTypesQuantil-QiskitRuntime_w1">
+    <NodeType name="QiskitRuntime_w1" abstract="no" final="no" targetNamespace="https://ust-quantil.github.io/nodetypes" winery:bordercolor="#f51700">
+        <winery:PropertiesDefinition elementname="QiskitRuntime_w1" namespace="https://ust-quantil.github.io/nodetypes">
+            <winery:properties>
+                <winery:constraints/>
+                <winery:defaultValue></winery:defaultValue>
+                <winery:description></winery:description>
+                <winery:key>RuntimeProgramID</winery:key>
```

Comment:
Shouldn't this be part of the program?

Reply:
+1
Otherwise we could not host multiple programs

Reply:
The Program-ID gets assigned by IBM and we just output it. How can we handle the case where there are multiple Programs and we want to output all their program-ids? Is there a way to make these keys dynamic (i.e. Program1ID, Program2ID,...) for multiple Programs?

Reply:
Thats the reason why it makes sense to move it to the QiskitRuntimeProgram. Because then you model multiple QiskitRuntimePrograms all hosted on the same QiskitRuntime and they all bring their own DA with the Python files and their own property to store the program ID